jenkins-master/hooks/test

```diff
@@ -7,3 +7,6 @@ docker run --name cx-jenkins-master-test --detach --publish 8080:8080 \
 
 # Wait up to 5 minutes for Jenkins to be up and fail if it does not return 200 after this time
 docker exec cx-jenkins-master-test timeout 300 bash -c "until curl --fail http://localhost:8080/api/json; do sleep 5; done"
+
+# Check for 'SEVERE' messages in Jenkins log (usually caused by plugin or configuration issues)
+docker logs cx-jenkins-master-test 2> >(grep --quiet SEVERE)
```

Comment:
What is the expected behavior here?

Reply:
Non-zero exit code if the term `SEVERE` is in the log. This happens for example when invalid plugin configuration is present. In consequence, the test will fail if the script has a non-zero exit code.

Reply:
Do you have a link to a working instance? I couldn't execute the command with the given syntax.

Reply:
Yes, replace the plugins.txt with the snippet above and you can reproduce the error condition.